README.md

```diff
@@ -63,39 +63,40 @@ All indicators will be available in window object. So you can just use
 sma({period : 5, values : [1,2,3,4,5,6,7,8,9], reversedInput : true});
 ```
 
-or 
+or
 
 ``` javascript
 SMA.calculate({period : 5, values : [1,2,3,4,5,6,7,8,9]});
 ```
 
 
 # Playground
-  
+
 [Playground with code completion](http://anandanand84.github.io/technicalindicators/ "Playground")
 
 
 # Available Indicators
 
-1. [Simple Moving Average (SMA)](https://tonicdev.com/anandaravindan/sma "SMA").
-1. [Exponential Moving Average (EMA)](https://tonicdev.com/anandaravindan/ema "EMA").
-1. [Weighted Moving Average (WMA)](https://tonicdev.com/anandaravindan/wma "WMA").
-1. [Moving Average Convergence Divergence (MACD)](https://tonicdev.com/anandaravindan/macd "MACD").
-1. [Bollinger Bands (BB)](https://tonicdev.com/anandaravindan/bb "BB").
+1. [Accumulation Distribution Line (ADL)](https://tonicdev.com/anandaravindan/adl "ADL").
+1. [Average Directional Index (ADX)](https://github.com/anandanand84/technicalindicators/blob/master/test/directionalmovement/ADX.js "ADX").
 1. [Average True Range (ATR)](https://tonicdev.com/anandaravindan/atr "ATR").
-1. [Relative Strength Index (RSI)](https://tonicdev.com/anandaravindan/rsi "RSI").
-1. [Wilder’s Smoothing (Smoothed Moving Average, WEMA)](https://tonicdev.com/anandaravindan/wema "WEMA").
-1. [Rate of Change (ROC)](https://tonicdev.com/anandaravindan/roc "ROC").
+1. [Bollinger Bands (BB)](https://tonicdev.com/anandaravindan/bb "BB").
+1. [Commodity Channel Index (CCI)](https://github.com/anandanand84/technicalindicators/blob/master/test/oscillators/CCI.js "CCI").
+1. [Force Index (FI)](https://github.com/anandanand84/technicalindicators/blob/master/test/volume/ForceIndex.js "VWAP").
 1. [Know Sure Thing (KST)](https://tonicdev.com/anandaravindan/kst "KST").
-1. [Stochastic Oscillator (KD)](https://tonicdev.com/anandaravindan/stochastic "KD").
-1. [WilliamsR (W%R)](https://tonicdev.com/anandaravindan/williamsr "W%R").
-1. [Accumulation Distribution Line (ADL)](https://tonicdev.com/anandaravindan/adl "ADL").
+1. [Moving Average Convergence Divergence (MACD)](https://tonicdev.com/anandaravindan/macd "MACD").
 1. [On Balance Volume (OBV)](https://tonicdev.com/anandaravindan/obv "OBV").
+1. Parabolic Stop and Reverse (PSAR)
```

Comment:
I don't have any link here so was unsure what would be desirable.

Reply:
I added link to the test so they can look into the api, inputs and so on.